custom_command_logic/pso2/itemname.js

```diff
@@ -14,12 +17,35 @@ export class itemname extends BaseCommand {
      * @param message {Message}
      */
     execute(args, message) {
-        message.reply("Test");
+        let embed = {
+            color: 3447003,
+            title: "Search results:",
+            fields: []
+        }
+        let replyFunction;
+        if(message.guild) {
+            replyFunction = function(m) {message.reply(m);};
```

Comment:
Please avoid replying and mentioning users if not really necessary!

Reply:
Fixed in the latest commit.